### PR TITLE
Align with DBus terminology

### PR DESCRIPTION
**Pull request by oranja**
I spent some time trying to register a new listener and while doing so I realized that even when I had all the DBus info that I needed, it was not so straightforward to infer how to make it fit into `SIGNALS` and `INTERFACES` here.
After spending some time with the code, the [`dbus-next` docs](https://python-dbus-next.readthedocs.io/en/latest/) and getting everything to work, I thought that things would be made much easier if we changed the names here a little bit to match the DBus terminology:
* What was defined under `interface` is really the Service name.
* What was named the `proxy_interface` is really just the name of the Interface.

It's clear to see why these two are confusing: Both use the _reverse domain name notation_ (e.g. `org.freedesktop.login1`), and services often expose at their top level an interface with the same name as the service itself (as with `org.freedesktop.Notifications`).

Bottom line, these two commits make mostly cosmetic changes to `dbus.py`, but should help other contributors find their arms and legs more easily.

### CODE REVIEW

**Comment by muniter** [comment]
Thank you very much, yes I found dbus stuff pretty convoluted, this indeed makes things more clear, and concise. Thank you again!
